src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -3,24 +3,42 @@
 import * as ecc from '@bitcoin-js/tiny-secp256k1-asmjs';
 import { BIP32Factory } from 'bip32';
 import { payments, networks } from 'bitcoinjs-lib';
-import type { IHdWalletAccount } from '@/types';
-import { MAXIMUM_ACCOUNTS_TO_DISCOVER } from '@/constants';
+import BigNumber from 'bignumber.js';
+import type { IHdWalletAccount, ICoin } from '@/types';
```

Comment:
Please split this line as in few other PRs the number of types are growing. This way we would skip some conflicts.